recipes/r-choroplethrzip/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '1.5.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-choroplethrzip
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/arilamstein/choroplethrZip
```

Comment:
See this comment https://github.com/conda-forge/staged-recipes/pull/21266/files#r1032930508